tests/OutputFormatter/GithubActionsOutputFormatterTest.php

```diff
@@ -77,7 +77,7 @@ public function finishProvider(): iterable
             ],
             'errors' => [],
             'warnings' => [],
-            "::error file=/home/testuser/originalA.php,line=12::ACME\OriginalA must not depend on ACME\OriginalB (LayerA on LayerB)\n",
+            "::error file=/home/testuser/originalA.php,line=12::ACME\OriginalA must not depend on ACME\OriginalB (LayerA on LayerB)". PHP_EOL,
```

Comment:
I would have preferred the hardcoded `\n` but since the messages seem to be generated via symfony components, we mimic the EOL style of symfony here